app/invocation/invocation_action_card.tsx

```diff
@@ -100,6 +99,13 @@ export default class ActionCardComponent extends React.Component<Props, State> {
     );
   }
 
+  getCacheAddress() {
```

Comment:
can you update this function to also append the remote instance name (after the port number in the cache address?)